File: remote/src/main/java/com/kelsos/mbrc/annotations/Connection.java
package com.kelsos.mbrc.annotations;

import android.support.annotation.IntDef;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

public class Connection {
  public static final int OFF = 0;
  public static final int ON = 1;
  private Connection() {
    //no instance
  }

  @Retention(RetentionPolicy.SOURCE)
  @IntDef({
      OFF,
      ON
  })
  public @interface Status {

  }
}

File: mbrc-lib/src/main/kotlin/com/kelsos/mbrc/annotations/Connection.kt
package com.kelsos.mbrc.annotations

import android.support.annotation.IntDef

object Connection {
    const val OFF = 0L
    const val ON = 1L

    @kotlin.annotation.Retention(AnnotationRetention.SOURCE)
    @IntDef(OFF, ON)
    annotation class Status
}
